### DISCUSSION TITLE
ArchR doublet detection adjustment for homotypic doublets?

### DISCUSSION THREAD

**Discussion by RegnerM2015** [GreenleafLab/ArchR#570] [Feature Requests]
Very cool R package and great documentation! 

I wanted to clarify ArchR's doublet detection approach. I understand ArchR finds heterotypic doublets, as most methods do. However, it is not quite clear in the documentation whether addDoubletScores() adjusts for the proportion of homotypic doublets. 

DoubletFinder implements an optional procedure to adjust for the proportion of homotypic doublets based on how many "cell types" exist in the data set (https://github.com/chris-mcginnis-ucsf/DoubletFinder). 

My question is whether ArchR's addDoubletScores() function implements a similar procedure "under the hood"?

**Comment by jgranja24** [comment]
Hi @RegnerM2015,

Currently, ArchR does not do this, but I am interested in looking into this adjustment. This seems like a post-hoc adjustment rather would be an option to `filterDoublets` rather than `addDoubletScores`. Does that seem reasonable?

Best

Jeff

**Comment by RegnerM2015** [comment]
I agree this adjustment seems more appropriate in  `filterDoublets` rather than `addDoubletScores`

Thanks for clarifying!
